pulp_container/app/tasks/sync_stages.py

```diff
@@ -126,20 +160,42 @@ async def run(self):
                     list_dc = self.create_tagged_manifest_list(
                         tag_name, saved_artifact, content_data
                     )
-                    await self.put(list_dc)
-                    tag_dc.extra_data["tagged_manifest_dc"] = list_dc
+                    at_least_one_manifest = False
                     for manifest_data in content_data.get("manifests"):
                         man_dc = self.create_manifest(list_dc, manifest_data)
+                        if signature_source is not None:
+                            man_sig_dcs = await self.create_signatures(man_dc, signature_source)
+                            if self.signed_only and not man_sig_dcs:
+                                # do not pass down the pipeline unsigned manifests
```

Comment:
It should not cause any problems with association with tags or manifest lists within the pipeline because we are not passing the dc further.
As for admins, it might be friendly to log that certain manifests have been skipped due to absence of a signature. I'll add it at info level but only here, for per-arch images. Is that what you had in mind?